conf/.env.docker

```diff
@@ -32,3 +32,5 @@ REDIS_PORT="{{REDIS_PORT}}"
 REDIS_PASSWORD="{{REDIS_PASSWORD}}"
 
 GITHUB_TOKEN="{{GITHUB_TOKEN}}"
+
+MAX_CHILDREN="{{MAX_CHILDREN}}"
```

Comment:
If we need to adjust other NGINX tuning settings in the future , it would be nice to prefix these variables so they are like so: `NGINX_MAX_CHILDREN="{{NGINX_MAX_CHILDREN}}" `